yang/frr-zebra.yang

```diff
@@ -2965,4 +2965,161 @@ module frr-zebra {
     }
     // End of operational / state container
   }
+
+  /*
+   * Dataplane provider state information
+   */
+  grouping dplane-provider-state {
```

Comment:
should this "grouping" be outside the zebra block, and then included in the "container state" block?